webroot/adm/oncall/generate-cstg-keypair.html

```diff
@@ -0,0 +1,636 @@
+                                                    id="updateSite-workflow-afterSearch-name"
+                                                    name="updateSite-workflow-afterSearch-name" disabled readonly>
+                                                <div class="py-2">
+                                                    <label for="updateSite-workflow-afterSearch-domainNames">Domain Names (comma-separated list):</label>
+                                                    <input type="text" class="form-control"
+                                                        id="updateSite-workflow-afterSearch-domainNames"
+                                                        name="updateSite-workflow-afterSearch-domainNames">
+                                                </div>
+                                                <div class="py-2">
+                                                    <label for="updateSite-workflow-afterSearch-appNames">App Names (comma-separated list):</label>
+                                                    <input type="text" class="form-control"
+                                                        id="updateSite-workflow-afterSearch-appNames"
+                                                        name="updateSite-workflow-afterSearch-appNames">
+                                                </div>
+                                                <div class="py-2">
+                                                    <label for="updateSite-workflow-afterSearch-description">Description (Optional):</label>
+                                                    <input type="text" id="updateSite-workflow-afterSearch-description"
+                                                        class="form-control"
+                                                        name="updateSite-workflow-afterSearch-description">
+                                                </div>
+
+                                                <div class="form-check py-2">
+                                                    <input type="checkbox"
+                                                        id="updateSite-workflow-afterSearch-PUBLISHER"
+                                                        class="form-check-input"
+                                                        name="updateSite-workflow-afterSearch-clientTypes[]"
+                                                        value="PUBLISHER">
+                                                    <label for="updateSite-workflow-afterSearch-PUBLISHER">Is this participant a publisher? Only publishers are allowed to use CSTG.  </label>
+                                                    <div>If the participant is an Advertiser, check <a href="https://atlassian.thetradedesk.com/confluence/display/UID2/UID2+CSTG+How+to+create+a+new+client+side+keypair" target="_blank">UID2 CSTG How to create a new client side keypair</a> for details</div>
```

Comment:
Should we also have this when creating a new site?

Reply:
Added